src/architecture/services.md

```diff
@@ -457,10 +457,10 @@ fn private_api_handler(&self, context: &ApiContext)
 }
 ```
 
-`public_api_handler` and `private_api_handler` provide hooks for defining
-public and private API endpoints respectively using [Iron framework][iron].
-These methods receive an API context, which allows to read information from
-the blockchain, and to translate POST requests into Exonum transactions.
+`wire_api` provide hooks for defining public and private API endpoints
+respectively using [`actix-web`][actix-web].
```

Comment:
I think this could be better:

> `wire_api` provides hooks for defining public and private API endpoints of the service.

`actix-web` seems like an implementation detail in this context.